### DISCUSSION TITLE
Feature request: Possibility to specify timezone for data

### DISCUSSION THREAD

**Discussion by SvenRuin** [grafana/grafana-csv-datasource#189] [Ideas]
As I understand the reply on https://github.com/marcusolsson/grafana-csv-datasource/discussions/78, UTC is assumed when timestamps are parsed on the server, if they have no timezone information.

I would suggest to add the possibility to specify timezone for the datasource, when the timestamps have no timezone information.

In our case the data comes from dataloggers, where the timezone used is normally UTC+1, but this is not shown in the timestamps (and we have no possibility to change that). We don't use daylight saving time (but someone else might use that, which could be more complicated to handle).

**Comment by marcusolsson** [comment]
I think it should be a relatively easy thing to add, though I don't know when I'll be able to work on this. If anyone wants to have a go at this, I'd be happy to review the pull request.
